### PR TITLE
Add pull request template

### PR DESCRIPTION
**Pull request by kxepal**

### CODE REVIEW

**Comment by benkeen** [comment]
Nice! Could we also update this file to mention this doc? https://github.com/apache/couchdb-fauxton/blob/master/CONTRIBUTING.md

**Comment by kxepal** [comment]
How to update? Github already provides a link to CONTRIBUTING within the notice block.

**Comment by benkeen** [comment]
Oh sorry, perhaps I misunderstood? That doc has a section about `Commit messages should follow the following style:` with an example of what the commit should contain. Doesn't this PR contain different (better) content about what should be in the commit/PR message?

**Comment by kxepal** [comment]
Hm...I got your idea. Indeed, some stuff worth to put in Contributing (same RTC policy and else). The other bits are a helping reminder what to not forget to do since CONTRIBUTING file is a thing that will be read only few times when you make a first PR. If CONTRIBUTING content get changed, you may never know about. It's easy to forget some details after all.

I'll rethink this PR tomorrow. Thanks for the feedback! (:

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+<!-- Describe how we can test your changes. -->
+
+## Screenshot
+
+<!-- If visual or design changes were made, please provide the screenshots
+     of how it was and how it looks now in the way to easy detect the positive
+     improvements. -->
+
+## JIRA issue number
```
Comment (garrensmith) [inline .github/PULL_REQUEST_TEMPLATE.md]: We going to use Github issues now. Could you change this to Github issue

Comment (kxepal) [inline .github/PULL_REQUEST_TEMPLATE.md]: Is that legal now? And I don't see issues tab around.

Comment (garrensmith) [inline .github/PULL_REQUEST_TEMPLATE.md]: Yes it is. Just waiting for this ticket https://issues.apache.org/jira/browse/INFRA-13627 for it to be enabled.

Comment (kxepal) [inline .github/PULL_REQUEST_TEMPLATE.md]: Cool!

**Comment by garrensmith** [comment]
I'm closing since we have the standard CouchDB one now

**Pull request closed without merging**